dremio_toolkit/env_file_writer.py

```diff
@@ -25,7 +25,7 @@
 
 class EnvFileWriter:
     @staticmethod
-    def save_dremio_environment(env_def: EnvDefinition, output_file: str, datetime_utc: datetime) -> None:
+    def save_dremio_environment(env_def: EnvDefinition, output_file: str) -> None:
```

Comment:
previous version was a better solution in my opinion, by passing datetime_utc we can emulate/mock the timestamp for tests, without it we will always get the present timestamp during the tests.